db/migrate/20210527131318_create_directory_columns.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class CreateDirectoryColumns < ActiveRecord::Migration[6.1]
+  def up
+    create_table :directory_columns do |t|
+      t.string :name, null: true
+      t.integer :automatic_position, null: true
+      t.string :icon, null: true
+      t.integer :user_field_id, null: true
+      t.boolean :automatic, null: false
+      t.boolean :enabled, null: false
+      t.integer :position, null: false
+      t.datetime :created_at, default: -> { 'CURRENT_TIMESTAMP' }
+    end
+
```

Comment:
I am always suspicious of tables without indexes. I know this table is likely to be small but we should add indexes anyway. I recommend `enabled, position`, but if it's ever filtered by `user_field_id` that might be a good idea too.